highfidelity/hifi
# InteractiveWindow scripting API

## QA test
1. be in desktop mode
2. run script test  `scripts/developer/test/interactiveWindowTest.js`
3. a native window should pop up with the text in title bar "Native Desktop Window", and in center of the window you should see "OverlayWindowTestString"
4. switch to VR mode
5. the native window should disappear from the desktop and re-appear on the Virtual desktop with title "Virtual Desktop Window".
6. switch back to Desktop mode. Verify that you see the same result as in step 3.
7. stop the script. -> the window should disappear.
8. smoke test other already existing UI elements/windows.

## ~known issue(s)~
~- interactiveWindow content turns white when resizing main interface window OSX.~
